### PR TITLE
Highlighting current user & Support for tourist title

### PR DESCRIPTION
**Pull request by Piyush-Goenka**
1. Highlighting the current user's row: When a user is logged in and their name appears in the hacks standings table, their row will be highlighted for easy identification.
2. Support for Tourist title: Users with the title "Tourist" will now be supported, and this title will be displayed appropriately in the standings.
I have checked the changes made in the extension via using developer mode in chrome extension.
I have checked the highlighting part for a random user and it is working finely.
Screenshot 2024-10-09 at 7 10 22 PM

### CODE REVIEW

File: src/content.js [hunk 1]
```diff
@@ -23,6 +23,8 @@ const colors = {
   "Международный гроссмейстер": "#ff0000",
   "Legendary Grandmaster": "#ff0000",
   "Легендарный гроссмейстер": "#ff0000",
+  Tourist: "#ff0000",
```
Comment (rahulharpal1603) [inline src/content.js:26]: The tourist title has only the first letter as red, the other letters are black. See the code to handle legendary grandmasters. You have to code that part too. The highlight user row works good.

**Review by rahulharpal1603 — CHANGES_REQUESTED** [review]

**Comment by rahulharpal1603** [comment]
@Piyush-Goenka please make the changes requested above.

**Comment by Piyush-Goenka** [comment]
Hey @rahulharpal1603 
I have made the required changes that has been asked in the latest commit.
I have changed the first letter of tourist to red and rest to black.
Thanks for guiding me.

**Review by rahulharpal1603 — APPROVED** [review]

**Comment by rahulharpal1603** [comment]
Thank you! @Piyush-Goenka

**Pull request merged**